gen/jsonschema/schemas/ArtifactVerificationOptions.schema.json

```diff
@@ -155,11 +155,6 @@
             "additionalProperties": false,
             "type": "object",
             "oneOf": [
-                {
-                    "required": [
-                        "raw_bytes"
-                    ]
-                },
```

Comment:
Flagging: I don't fully understand this codegen change -- `raw_bytes` was originally marked explicitly with `optional`, so there absolutely should never have been a `required` variant to begin with.

Reply:
That is strange, what happens if you generate jsonschema from scratch before this change? Is required still present? Just wondering if this is stale or something.

Reply:
Hmm, if I re-generate before this change the `required` comes back. My only guess here is that because it's wrapped within a `oneOf`, it has no effect (since it's still optional under the other `oneOf` variants).